app/src/tracing.h

```diff
@@ -147,44 +153,26 @@ namespace scitt
       client_request_id =
         ctx.rpc_ctx->get_request_header(CLIENT_REQUEST_ID_HEADER);
 
-      if (client_request_id.has_value())
-      {
-        // Validate client request id to avoid misinterpretation of the log,
-        // e.g. if it contains a space.
-        if (!std::regex_match(
-              client_request_id.value(), CLIENT_REQUEST_ID_REGEX))
-        {
-          client_request_id = std::nullopt;
-          SCITT_INFO("Code=InvalidInput Invalid client request id");
-          ctx.rpc_ctx->set_error(
-            HTTP_STATUS_BAD_REQUEST,
-            errors::InvalidInput,
-            "Invalid client request id.");
-          return;
-        }
-        ctx.rpc_ctx->set_response_header(
-          "x-ms-client-request-id", client_request_id.value());
-      }
-
       // The user data is used to propagate the request IDs to the local
       // commit callback
       AppData& app_data = get_app_data(ctx.rpc_ctx);
       app_data.request_id = request_id;
       app_data.client_request_id = client_request_id;
 
-      auto query = ctx.rpc_ctx->get_request_query();
-
       SCITT_INFO(
-        "Verb={} Path={} URL={}",
+        "::START:: Verb={} Path={} Query={} URL={}",
         ctx.rpc_ctx->get_request_verb().c_str(),
-        method,
+        path,
+        ctx.rpc_ctx->get_request_query().c_str(),
         ctx.rpc_ctx->get_request_url());
 
       ::timespec start;
       ccf::ApiResult result = get_time(app_data.start_time);
       if (result != ccf::ApiResult::OK)
       {
-        SCITT_FAIL("Code=InternalError get_untrusted_host_time_v1 failed");
+        // FIXME: should be CBOR or JSON
```

Comment:
Does that need to be done before this merges, or is it separate?

Reply:
As mentioned in another comment it could in theory fail on SCITT endpoints

Reply:
Replaced with explicit CBOR error